package.json

```diff
@@ -65,6 +66,7 @@
     "node": ">=6.0.0"
   },
   "dependencies": {
-    "@angular/forms": "^4.1.2"
+    "@angular/forms": "^4.1.2",
+    "@types/google-libphonenumber": "^7.4.11"
```

Comment:
This library is to be used for mobile number validations, I didn't add that method in this pull request, but I will contribute that aswell.